### PR TITLE
Changelog entries for v0.3

### PR DESCRIPTION
**Pull request by exbinary**
Went through and collected all our recent changes into the Changelog.
Would love some editorial eyes :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+* @phylum
+* @thestephenmarshall
+* @maricris-sn
+* @indiebrain
+* @grenewode
+* @evertoncunha
+* @eabousaif
+* @danielmmack
+* @cflipse
+* @acherukuri
```
Comment (exbinary) [inline CHANGELOG.md:44]: This is missing a bunch of RfG folks. Will update.

Comment (exbinary) [reply CHANGELOG.md:44]: Actually, AFAIC tell, other RfG folk's branches haven't hit `main` yet so can acknowledge them in an upcoming release.

**Review by maebeale — APPROVED** [review]
This is an amazing release!!! Thanks everyone!!! And thanks, @exbinary for going through to document.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,33 +1,89 @@
...
+This codebase isn't able to strictly adhere to the [Semantic Versioning Specification](https://semver.org/) because it doesn't currenty declare a public API.
+
+However, to support upgrade paths for mutual aid groups who've deployed the app, we will use common SemVer practices to communicate the impact of changes in each release.
+
+We're still in the [initial development phase](https://www.jering.tech/articles/semantic-versioning-in-practice#initial-development-phase), so we are versioning releases with these modified rules:
+* Major version (0.x.y) -> will remain at 0 until we're ready to release v1.
+* Minor version (0.X.y) -> will be incremented to indicate a **breaking** change, something that will require some action on your part in order to upgrade.
+* Patch version (0.x.Y) -> incremented whenever we release backward compatible changes, including new features, bugfixes etc.
+
+This changelog also serves to acknowledge the incredible people who've contributed brilliance, effort and being. Their handles are listed under the first release they each  touched. 💗🙏🏾
```
Comment (maebeale) [inline CHANGELOG.md:12]: ❤️ ❤️ ❤️ ❤️ ❤️ ❤️

**Pull request merged**